pkg/api/job_runs.go

```diff
@@ -114,28 +115,60 @@ func JobRunRiskAnalysis(dbc *db.DB, jobRun *models.ProwJobRun) (apitype.ProwJobR
 		compareRelease = ar[0].Release
 	}
 
+	// hack since we don't currently get the jobRunTestCount for 4.12 jobs.
+	// If the jobRunTestCount is 0 and we are pre 4.13 set the jobRunTestCount to the historicalCount
+	if jobRunTestCount <= 0 {
+		var major, minor int64
+		var err error
+		versions := strings.Split(compareRelease, ".")
+		if len(versions) > 0 {
+			major, err = strconv.ParseInt(versions[0], 10, 64)
+
+			if err != nil {
+				log.Errorf("Error parsing major version for %s: %v", compareRelease, err)
+			}
+
+			if len(versions) > 1 {
+				minor, err = strconv.ParseInt(versions[1], 10, 64)
+			}
+
+			if err != nil {
+				log.Errorf("Error parsing minor version for %s: %v", compareRelease, err)
+			}
+
+			// ignoring the errors above after logging since we are in an edge case
+			if major < 4 || (major == 4 && minor < 13) {
+				jobRunTestCount = historicalCount
+			}
+		}
+	}
+
 	// NOTE: we are including bugs for all releases, may want to filter here in future to just those
 	// with an AffectsVersions that seems to match our compareRelease?
 	jobBugs, err := query.LoadBugsForJobs(dbc, []int{int(jobRun.ProwJob.ID)}, true)
 	if err != nil {
-		return apitype.ProwJobRunRiskAnalysis{}, err
+		log.Errorf("Error evaluating bugs for prow job: %d", jobRun.ProwJob.ID)
+	} else {
+		jobRun.ProwJob.Bugs = jobBugs
 	}
-	jobRun.ProwJob.Bugs = jobBugs
 
 	// Pre-load test bugs as well:
 	if len(jobRun.Tests) <= maxFailuresToFullyAnalyze {
 		for i, tr := range jobRun.Tests {
 			bugs, err := query.LoadBugsForTest(dbc, tr.Test.Name, true)
 			if err != nil {
-				return apitype.ProwJobRunRiskAnalysis{}, err
+				// SELECT * FROM "tests" WHERE name = '[sig-arch][Feature:ClusterUpgrade] Cluster should remain functional during upgrade [Disruptive] [Serial]' ORDER BY "tests"."id" LIMIT 1;
+				// causes an error here, don't think we need to fail due to missing bugs - error(*errors.errorString) *{s: "record not found"}
```

Comment:
I can remove the commented queries but I'm curious about them and wanted to document / followup a little